keras_nlp/models/bert/bert_backbone.py

```diff
@@ -163,6 +163,7 @@ def __init__(
                     x, approximate=True
                 ),
                 dropout=dropout,
+                layer_norm_epilson=1e-12,
```

Comment:
Hey, @soma2000-lang. The tests are failing because `epsilon` has been misspelt as `epilson`. Could you please correct and push again? Thanks!